dnf/lock.py

```diff
@@ -77,8 +77,10 @@ def _try_lock(self):
             os.write(fd, pid)
             os.close(fd)
             return True
-        except OSError:
-            return False
+        except OSError as e:
+            if e.errno == 17:   # File exists
+                return False
+            raise
```

Comment:
what do you want to raise here? not sure if raising plain Exception is a good idea. If it is good idea - then write some message for it.